custom_components/versatile_thermostat/base_thermostat.py

```diff
@@ -134,6 +138,7 @@
 from .ema import ExponentialMovingAverage
 
 _LOGGER = logging.getLogger(__name__)
+ConfigData = Mapping[str, Any]
```

Comment:
Correction: This type should be `MappingProxyType`, not `Mapping`. More details in [another comment](https://github.com/jmcollin78/versatile_thermostat/pull/345#discussion_r1465733083). I am force-pushing this change.